src/api/.database_consistency.todo.yml

```diff
@@ -1718,6 +1718,10 @@ ReleaseTarget:
   id:
     PrimaryKeyTypeChecker:
       enabled: false
+Report:
+  reportable:
+    ColumnPresenceChecker:
+      enabled: false
```

Comment:
Why disabling instead of validating?

Reply:
Thinking twice... the issue is a different one. I don't have to fix or ignore the lack of consistency in the DB.

The problem is that we were making `Report#reportable` compulsory, but we shouldn't as we can nullify it at some point.

So I'm going to modify this line:
https://github.com/openSUSE/open-build-service/blob/eccd4b1163b1b6040f62bb6bb3cc533320428fd7/src/api/app/models/report.rb#L7

Reply:
done, @hennevogel @hellcp-work